### PR TITLE
Plumb libra accounts to genesis

### LINKED ISSUE #5332: No bench-tps via Move metrics

### PR DESCRIPTION
**Pull request by sakridge**
#### Problem

Real genesis needs to create libra mint accounts for bench-tps to work.

#### Summary of Changes

Add libra mint account to genesis creation.

Fixes #5332

### CODE REVIEW

File: genesis/src/main.rs [hunk 1]
```diff
@@ -147,6 +150,14 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                 .required(true)
                 .help("Path to file containing the bootstrap leader's storage keypair"),
         )
+        .arg(
+            Arg::with_name("libra_mint_keypair_file")
+                .long("libra-mint-keypair")
+                .value_name("LIBRA MINT KEYPAIR")
+                .takes_value(true)
+                .required(true)
```
Comment (mvines) [inline genesis/src/main.rs]: Libra keypair probably shouldn’t be required

File: core/tests/local_cluster.rs [hunk 2]
```diff
@@ -21,6 +21,7 @@ use std::time::Duration;
 #[serial]
 fn test_ledger_cleanup_service() {
     solana_logger::setup();
+    error!("test_ledger_cleanup_service");
```
Comment (mvines) [inline core/tests/local_cluster.rs]: Might as well just disable all local_cluster tests at this point, something is very broken

File: bench-tps/src/bench.rs [hunk 3]
```diff
@@ -234,50 +234,100 @@ fn metrics_submit_lamport_balance(lamport_balance: u64) {
...
+                    1,
+                    *blockhash,
+                ),
+                timestamp(),
+            )
+        })
+        .collect()
+}
+
+fn generate_solana_txs(
```
Comment (mvines) [inline bench-tps/src/bench.rs]: Suggest: generate_system_txs()

File: bench-tps/src/main.rs [hunk 4]
```diff
@@ -80,7 +82,30 @@ fn main() {
...
+                eprintln!(
+                    "{}: Unable to open keypair file: {}",
+                    err, libra_mint_keypair_file
+                );
+                exit(1);
+            }),
+        ))
+    } else {
+        if use_move {
+            eprintln!("Keypair file needed if move is enabled, use --libra-mint-keypair argument");
```
Comment (mvines) [inline bench-tps/src/main.rs]: You can get Clap to do this for you.  Just tell Clap that these two args require each other

File: genesis/src/main.rs (hunk 1 above)
Comment (sakridge) [reply genesis/src/main.rs]: Yes, this is the hack that I brought up on discord.

Comment (sakridge) [reply genesis/src/main.rs]: Ah, you mean a required argument. Yea it could be optional then don't add the libra account to genesis.

Comment (mvines) [reply genesis/src/main.rs]: Yeah that :)

File: core/tests/local_cluster.rs (hunk 2 above)
Comment (sakridge) [reply core/tests/local_cluster.rs]: Done in another PR.

**Pull request merged**